modules/core/02-client/keeper/grpc_query.go

```diff
@@ -328,3 +328,55 @@ func (k Keeper) UpgradedConsensusState(c context.Context, req *types.QueryUpgrad
 		UpgradedConsensusState: protoAny,
 	}, nil
 }
+
+// VerifyMembership implements the Query/VerifyMembership gRPC method
```

Comment:
should we add more godoc? (no state changes occur, we charge gas if possible, meant for state machine usage, delegates to light cilents)

Reply:
done in https://github.com/cosmos/ibc-go/pull/5821/commits/00f59e866d2d8df3e2aca3c31ce9c68340bd4763